### PR TITLE
Add stress test2

### PR DESCRIPTION
**Pull request by pillip8282**
It should be merged after fix_onoff_driver is merged.

### CODE REVIEW

File: os/net/netmgr/netdev_wifi.c [hunk 1]
```diff
@@ -30,6 +31,7 @@
 int netdev_handle_wifi(struct netdev *dev, lwnl_req cmd, void *data, uint32_t data_len)
 {
 	lwnl_result_e res = LWNL_FAIL;
+	lldbg("T%d cmd(%d) (%p) (%d)\n", cmd, data, data_len);
```
Comment (deepaksrma) [inline os/net/netmgr/netdev_wifi.c]: can we use ndbg here?

File: apps/examples/wifi_manager_sample/wm_test_on_off.c [hunk 2]
```diff
@@ -198,82 +202,158 @@ static void run_init(void *arg)
...
+	abstime.tv_sec  = before.tv_sec + WO_WAIT_CONN_TIME;
+	abstime.tv_nsec = before.tv_nsec;
+	printf("[WO] wait connect time %d \n", WO_WAIT_CONN_TIME);
+	int ret = sem_timedwait(&g_wm_sem, &abstime);
+	printf("[WO] [pkbuild] %s:%d\n", __FUNCTION__, __LINE__);
+	if (ret < 0) {
+		if (errno == ETIMEDOUT) {
+			printf("[WO] [pkbuild] %s:%d\n", __FUNCTION__, __LINE__);
+			printf("[WO] timeout \n");
+			// wifi doens't respond. so initialize agina
```
Comment (deepaksrma) [inline apps/examples/wifi_manager_sample/wm_test_on_off.c]: *again and *doesn't

**Review by an4967 — APPROVED** [review]
LGTM
It works well

**Review by sangwon03 — APPROVED** [review]
LGTM

**Review by an4967 — APPROVED** [review]
It works well.  
LGTM

**Review by sangwon03 — APPROVED** [review]
LGTM

**Pull request merged**